packages/db/tests/models/test-granule-model.js

```diff
@@ -652,10 +1840,58 @@ test('GranulePgModel.upsert() succeeds without an execution for completed granul
     collection_cumulus_id: collectionCumulusId,
   });
 
-  await granulePgModel.upsert(knex, granule);
+  await granulePgModel.upsert({ knexOrTrx: knex, granule });
   t.true(await granulePgModel.exists(knex, granule));
 });
 
+test('GranulePgModel.upsert() throws an error for a granule with status of "completed" without createdAt set when write constraints are set to true and sets a default value', async (t) => {
```

Comment:
These test additions are great! I see that the test covers `throws an error for a granule with status of "completed" without createdAt set when write constraints are set to true` but I don't see the check for setting a default value 🤔 I definitely could just be missing something.

Reply:
You're not, it's mis-titled, probably due to copy/paste.   It cannot set a default value if it throws/fails to write.